scripts/twister

```diff
@@ -373,290 +458,293 @@ Artificially long but functional example:
 
     parser.add_argument(
-        "-G",
-        "--integration",
-        action="store_true",
-        help="Run integration tests")
-
-    case_select.add_argument(
-        "-F",
-        "--load-tests",
-        metavar="FILENAME",
-        action="store",
-        help="Load list of tests and platforms to be run from file.")
+        "-j", "--jobs", type=int,
+        help="Number of jobs for building, defaults to number of CPU threads, "
+             "overcommited by factor 2 when --build-only")
 
     parser.add_argument(
-        "--quarantine-list",
-        metavar="FILENAME",
-        help="Load list of test scenarios under quarantine. The entries in "
-             "the file need to correspond to the test scenarios names as in"
-             "corresponding tests .yaml files. These scenarios"
-             "will be skipped with quarantine as the reason")
+        "--json-report", action="store_true",
+        help="""Generate a JSON file with all test results. [Experimental]
+        """)
 
     parser.add_argument(
-        "--quarantine-verify",
-        action="store_true",
-        help="Use the list of test scenarios under quarantine and run them"
-             "to verify their current status")
-
-    case_select.add_argument(
-        "-E",
-        "--save-tests",
-        metavar="FILENAME",
-        action="store",
-        help="Append list of tests and platforms to be run to file.")
-
-    test_or_build = parser.add_mutually_exclusive_group()
-    test_or_build.add_argument(
-        "-b", "--build-only", action="store_true",
-        help="Only build the code, do not execute any of it in QEMU")
-
-    test_or_build.add_argument(
-        "--test-only", action="store_true",
-        help="""Only run device tests with current artifacts, do not build
-             the code""")
+        "-K", "--force-platform", action="store_true",
+        help="""Force testing on selected platforms,
+        even if they are excluded in the test configuration (testcase.yaml)"""
+    )
 
     parser.add_argument(
-        "--cmake-only", action="store_true",
-        help="Only run cmake, do not build or run.")
+        "-l", "--all", action="store_true",
+        help="Build/test on all platforms. Any --platform arguments "
+             "ignored.")
 
-    parser.add_argument(
-        "--filter", choices=['buildable', 'runnable'],
-        default='buildable',
-        help="""Filter tests to be built and executed. By default everything is
-        built and if a test is runnable (emulation or a connected device), it
-        is run. This option allows for example to only build tests that can
-        actually be run. Runnable is a subset of buildable.""")
+    parser.add_argument("--list-tags", action="store_true",
+                        help="list all tags in selected tests")
 
+    parser.add_argument("--log-file", metavar="FILENAME", action="store",
+                        help="log also to file")
```

Comment:
This msg is too cryptic.  "Specify a file where to save logs." ?

Reply:
ok